<launch>

  <arg name="model" default="$(find wpr1_bringup)/urdf/wpr1.urdf"/>
  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  <arg name="gui" default="true" />
  <arg name="rvizconfig" default="$(find wpr1_bringup)/rviz/urdf.rviz" />
  <param name="use_gui" value="$(arg gui)"/>

  <!-- rviz -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

  <!-- wpr1 wpr1_mani_sim-->
  <node pkg="wpr1_bringup" type="wpr1_mani_sim" name="wpr1_mani_sim" output="screen">
  </node>

</launch>